ONSdigital/ssdc-rm-ddl
# 1288 upgrade spring boot version in java repos

# Has the DDL schema changed?
*Pull requests which include schema changes should be labelled with the `schema change` label for visibility*

*The [common entity model pom.xml](ssdc-rm-common-entity-model/pom.xml) file `project.version` must be bumped
appropriately if there are any common entity code changes*

  version: 

# Motivation and Context

Spring Boot 3 has been out for a while and Dependabot wants us to upgrade to it. There's quite a few changes which you look up for yourselves. I listed main changes to each repo on the ticket.

# What has changed




# How to test?



- Before running this branch, Run `make build` on the exception manager, uac-qid service, ssdc-rm-shared-sample-validation repos. These need to be built because they get passed into the ddl script
- Runn `UAC_QID_SERVICE_BRANCH=1288-upgrade-spring-boot-version-in-java-repos EXCEPTION_MANAGER_BRANCH=1288-upgrade-spring-boot-version-in-java-repos make dev-build
`. This should build the expected ddl changes for you.
- run `make build` on the rest of the java services
- Run the ATs and make sure everything works as expected.
# Links



[Trello](https://trello.com/c/Oo7EZ4q4/)
# Screenshots (if appropriate):
